src/backend/gpopt/translate/CTranslatorDXLToPlStmt.cpp

```diff
@@ -4724,6 +4728,16 @@ CTranslatorDXLToPlStmt::TranslateDXLTblDescrToRangeTblEntry(
 
 	rte->eref = alias;
 
+	/*
+		Store the alias information only when table alias is used to allow 
+		EXPLAIN find it
+	*/
+	if (optimizer_enable_table_alias && NULL != md_alias)
```

Comment:
Why is this check needed?! The md_alias pointer is dereferenced above, which means it is definitely not NULL here!